backend/inspirehep/records/api/base.py

```diff
@@ -477,3 +537,27 @@ def revert(self, revision_id):
 
         after_record_revert.send(current_app._get_current_object(), record=self)
         return self
+
+    @property
+    def control_number(self, pid=False):
```

Comment:
what is the `pid` flag used for?

Reply:
Leftover from time when this was not a property ;) . Sorry.